controllers/pxc/transition_key_copier.go

```diff
@@ -0,0 +1,106 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	pxcv1 "github.com/Percona-Lab/k8s-vault-issuer/apis/pxc/v1"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *PerconaXtraDBClusterReconciler) processTransferVaultKeys(currClusterCR *pxcv1.PerconaXtraDBCluster, clusters []string) error {
+	currClusterName := currClusterCR.Name + "." + currClusterCR.Namespace
+	for _, v := range clusters {
+		splittedName := strings.Split(v, ".")
+		if len(splittedName) != 2 {
+			return errors.New("Invalid source cluster name, plaease use format clusterName.namespace")
```

Comment:
Should not be capitalized.
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
1b5a6ce29dd0093ba31f62f1ad48e462f6b89c92